### PR TITLE
[SPARK-50507][PYTHON][TESTS][FOLLOW-UP] Add refactored package into pure Python test

### PR DESCRIPTION
**Pull request by HyukjinKwon**
### What changes were proposed in this pull request?

This PR is a followup of https://github.com/apache/spark/pull/49074 that adds refactored package into pure Python test

### Why are the changes needed?

In order to fix the pure Python build https://github.com/apache/spark/actions/runs/12215379954/job/34077255570.

### Does this PR introduce _any_ user-facing change?

No, test-only.

### How was this patch tested?

Will monitor the build.

### Was this patch authored or co-authored using generative AI tooling?

No.

### CODE REVIEW

**Review by itholic — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

File: python/packaging/connect/setup.py [hunk 1]
```diff
@@ -101,6 +101,7 @@
         "pyspark.pandas.tests.connect.groupby",
         "pyspark.pandas.tests.connect.indexes",
         "pyspark.pandas.tests.connect.io",
+        "pyspark.pandas.tests.connect.pandas",
```
Comment (ueshin) [inline python/packaging/connect/setup.py:104]: Shouldn't this be: `pyspark.sql.tests.connect.pandas`?

Comment (HyukjinKwon) [reply python/packaging/connect/setup.py:104]: yes :-). making a pr now

Comment (HyukjinKwon) [reply python/packaging/connect/setup.py:104]: https://github.com/apache/spark/pull/49119

**Pull request closed without merging**